src/test/java/org/sonarsource/sonarlint/ls/settings/SettingsManagerTests.java

```diff
@@ -795,6 +795,24 @@ void shouldIgnoreRazorFiles() {
     assertThat(analyzerProperties).contains(entry("sonar.cs.file.suffixes", ".cs"));
   }
 
+  @Test
+  void shouldIgnoreRazorCsFiles() {
+    var workspaceUri = URI.create("file:///User/user/documents/project");
+    var sonarLintSettings = new JsonObject();
+    sonarLintSettings.add("disableTelemetry", new JsonPrimitive(false));
+    sonarLintSettings.add("focusOnNewCode", new JsonPrimitive(true));
```

Comment:
Why are these 3 lines duplicated in a few test cases in this class? It seems that the test cases run fine without them anyway, and I find suspicious that only these 2 properties are changed. If anything, `disableTelemetry` should be `true` and `focusOnNewCode` should probably be `false` in tests.

Reply:
Indeed, setting the telemetry and new code settings are not needed. I removed the two lines. The rest is necessary in order to properly set the analyzer properties, I think.